src/OmniSharp.Abstractions/Extensions/LoggerExceptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Microsoft.Extensions.Logging
+{
+    public static class LoggerExceptions
+    {
+        public static void LogError(this ILogger logger, Exception ex, string message, params object[] args)
```

Comment:
Yeah, I hate the event ID thing. Should be optional. There's no reason to have a "0" everywhere if you aren't using them. :smile: